terraform/main.tf

```diff
@@ -61,15 +60,15 @@ module "backoffice-alb" {
   private_subnets = module.vpc.private_subnets
 
   lb_ssl_policy       = var.lb_ssl_policy
-  ssl_certificate_arn = var.ssl_certificate_arn
+  ssl_certificate_arn = module.acm.ssl_certificate_arn
```

Comment:
I'm worried what this will look like when we try to cut across. Should we be keeping the old certs until we have the dns delegated to us?

Reply:
Good spot. I've reversed this for now.